packages/list/README.md

```diff
@@ -108,29 +109,37 @@ class MyApp extends Component {
     );
   }
 }
+
 ```
 
 ### List groups and list dividers
 
 Multiple related lists can be grouped together using the `ListGroup` component. Optional subheaders can be added using `ListGroupSubheader`. `ListDivider`s can be used to separate content either within a list or between lists.
 
 ```js
-import React, {Component} from 'react';
-import List, {ListItem, ListItemText, ListGroup, ListGroupSubheader} from '@material/react-list';
+import React, { Component } from "react";
+import List, {
+  ListItem, ListItemText, ListGroup, 
+  ListGroupSubheader,ListDivider
+} from "@material/react-list";
 
 class MyApp extends Component {
   render() {
     return (
       <ListGroup>
-        <ListGroupSubheader tag='h2'>Folders</ListGroupSubheader>
+        <ListGroupSubheader tag="h2">Folders</ListGroupSubheader>
```

Comment:
single quotes please, and the instances below.

Reply:
Done